databases/Spider.yaml

```diff
@@ -0,0 +1,16 @@
+database_name: spider_examples
+sqlalchemy_uri: spider://
+cache_timeout: null
+expose_in_sqllab: true
+allow_run_async: false
+allow_ctas: false
+allow_cvas: false
+allow_dml: false
+allow_csv_upload: false
+extra:
+  metadata_params: {}
+  engine_params: {}
+  metadata_cache_timeout: {}
+  schemas_allowed_for_csv_upload: []
+uuid: 4a8e60d1-307f-aa55-9a38-894f2ee7c58b
```

Comment:
Is this randomly generated?

Reply:
I generated it using my new BFF ChatGPT. I also could generate one "in superset" by manually  connecting the DB and exporting it so I can get a `uuid` from it if that's preferred.